include/fiddle/postprocess/fiber_network.h

```diff
@@ -0,0 +1,64 @@
+#ifndef included_fiddle_postprocess_fiber_network_h
+#define included_fiddle_postprocess_fiber_network_h
+
+#include <deal.II/distributed/shared_tria.h>
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/grid/tria.h>
+
+namespace fdl
+{
+  using namespace dealii;
+    
+    template <int dim, int spacedim = dim>
+    class FiberNetwork
+    {
+        /*
+        stores several cell-centered fiber network data
+        */
+    public:
+
+        FiberNetwork(const Triangulation<dim, spacedim> &tria,
+                    const std::vector<std::vector<Tensor<1, spacedim>>> &fibers)
+                    : tria(&tria)  
+                    {

```

Comment:
This function should be in the source file too. Check out the other source files to see how we handle explicit instantiations (even though it is a template class we want to define it in source files and only instantiate for valid dim/spacedim pairs).